File: FrameIncam.Domains/Common/PaymentOrderModel.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace FrameIncam.Domains.Common
{
    public class PaymentOrderModel
    {
        public string order_id { get; set; }
        public string key { get; set; }
        public int amount { get; set; }
        public string currency { get; set; }
        public string name { get; set; }
        public string email { get; set; }
        public string contactNumber { get; set; }
        public string address { get; set; }
        public string description { get; set; }

        public Prefill prefill { get; set; }
    }
    public class Prefill
    {
        public string name { get; set; }
        public string contact { get; set; }
        public string email { get; set; }
    }
}

File: FrameIncam.Client/ng/domains/common/paymentOrderModel.js
(function ($, ng, debugMode, undefined) {
    "use strict";

    // --------------------------------------------------------------------------------
    // Dependencies
    // --------------------------------------------------------------------------------
    var module = ng.module("app");

    // --------------------------------------------------------------------------------
    // Model
    // --------------------------------------------------------------------------------
    module.run([
        "services.utils",
        "services.models",
        function (
            p_utils,
            p_models
        ) {
            // --------------------------------------------------------------------------------
            // Namespace
            // --------------------------------------------------------------------------------
            var fincam = p_models.namespace("fincam");
            var common = p_models.namespace("fincam.common");
            var config = p_models.namespace("fincam.config");
            var commonSubscription = p_models.namespace("fincam.common.subscription");
            // --------------------------------------------------------------------------------
            // Model
            // --------------------------------------------------------------------------------
            p_models.class({
                namespace: common,
                name: "paymentOrderModel",
                base: {
                    namespace: fincam,
                    name: "model"
                },
                properties: {
                    order_id: {
                        name: "string"
                    },
                    key: {
                        name: "string"
                    },
                    amount: {
                        name: "int"
                    },
                    currency: {
                        name: "string"
                    },
                    name: {
                        name: "string"
                    },
                    email: {
                        name: "string"
                    },
                    contactNumber: {
                        name: "string"
                    },
                    address: {
                        name: "string"
                    },
                    description: {
                        name: "string"
                    },
                    prefill:{
                        namespace: commonSubscription,
                        name:"prefill"
                    }
                },
                prototype: new function () {
                    var self = this;
                    return self;
                }
            });
        }
    ]);

})(jQuery, angular, document.children[0].hasAttribute("debug"));